napari/layers/points/_points_mouse_bindings.py

```diff
@@ -82,19 +82,20 @@ def select(layer, event):
     layer._set_highlight(force=True)
 
 
+DRAG_DIST_THRESHOLD = 5
+
+
 def add(layer, event):
     """Add a new point at the clicked position."""
-    # on press
-    dragged = False
-    yield
 
-    # on move
-    while event.type == 'mouse_move':
-        dragged = True
+    if event.type == 'mouse_press':
```

Comment:
Just curious is this `event.type == 'mouse_press'` necessary? I would have thought that based on how we handle the generators it's always true. Did you see an error or were you adding it as a precaution?